app/i18n/fr/conf.php

```diff
@@ -32,6 +32,7 @@
 	),
 	'display' => array(
 		'_' => 'Affichage',
+		'darkMode' => 'Automatically dark mode (Beta)',	// TODO
```

Comment:
```suggestion
		'darkMode' => 'Mode sombre automatique (bêta)',
```

Reply:
The original should be `Automatic dark mode (beta)` btw.

Automatic for grammar, beta for style.